Java Timer Tasks

Joesongoma opened the pull request:
### Article checklist

#### Pre-submission checks
> Please ensure that you have only one open topic suggestion (issue) + in-review article (linked pull request) at a time. This will ensure that we complete the review process in a timely manner from inception to publishing

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [X] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [X] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [X] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [X] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [X] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [X] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [X] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [X] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [X] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [X] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [X] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [X] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [X] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [X] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [X] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [X] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes
> Remove the backticks and add the issue number below to link and close your Topic Suggestion (issue) when your article has been published (PR has been merged). See this [video](https://youtu.be/TKJ4RdhyB5Y?t=278) for more details.

`This closes #3868 

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

LinusMuema commented:
plagiarism check done :heavy_check_mark:

Joesongoma commented:
Hello @Jethro-magaji 
I have done the requested changes.

Jethro-magaji commented:
> Hello @Jethro-magaji I have done the requested changes.

Hi! follow this [link ](https://github.com/section-engineering-education/engineering-education/pull/4227/commits/6e1bd49781c1b15e6029e192a6827e9e598a41f5) to see some reviews I requested previously of which haven't been attended to... Please do that and let me know.
Thanks.

Joesongoma commented:
@Jethro-magaji 
Hi, I just can't understand, I made the changes you requested; removed all typos, corrected grammar, and I used the second person singular as you requested. Even the link you are giving me takes me to my last commit, so I don't really get what you want me change.

Jethro-magaji commented:
> @Jethro-magaji Hi, the link is opening my last commit

Here's the link 
https://github.com/section-engineering-education/engineering-education/pull/4227/files/29c9a15fbe449fc51b2a7406596a226f3631fba2

Joesongoma commented:
> @Joesongoma Good afternoon and thank you for submitting your topic to the EngEd program.
> 
> @Jethro-magaji If it seems this PR is still quite a bit aways from being completed lets go ahead and close it - to allow the author the time to make the required edits.
> 
> @Joesongoma - As a rule of thumb - please only submit articles (pull requests) with a complete and fully polished article - ready to be published.
> 
> All revisions and edits should be completed on your own forked repo (as to not build up a backlog in the queue). We will be closing this and please re-submit when it is ready to be published as a new PR (and you can add the duplicate label) and ping the reviewer and myself.

 @hectorkambow 
Hello, I did the changes as requested by the peer reviewer.

Joesongoma commented:
> @Joesongoma Good afternoon and thank you for submitting your topic to the EngEd program.
> 
> @Jethro-magaji If it seems this PR is still quite a bit aways from being completed lets go ahead and close it - to allow the author the time to make the required edits.
> 
> @Joesongoma - As a rule of thumb - please only submit articles (pull requests) with a complete and fully polished article - ready to be published.
> 
> All revisions and edits should be completed on your own forked repo (as to not build up a backlog in the queue). We will be closing this and please re-submit when it is ready to be published as a new PR (and you can add the duplicate label) and ping the reviewer and myself.

@hectorkambow 
It was an error I didn't notice and am fixing it.

Jethro-magaji commented on content/articles/java-timer-tasks/index.md:
```diff
@@ -0,0 +1,131 @@
+
+##### Conclusion
+
+This tutorial has taken the reader through the concept of Timer Tasks in Java, and therefore, the reader should be able to get a better understanding of Timer Tasks and create his or her own timer that alerts or displays a given message when time is up.
```
> Use 2nd person singular here too...

Hi @Joesongoma 
Please ignore some of these reviews as you have done them, just want to show you what I was talking about with these comments...

WanjaMIKE commented:
Merged #4227 into master
